patches0/patch-package+6.2.2.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/patch-package/dist/applyPatches.js b/node_modules/patch-package/dist/applyPatches.js
+index 35a22f7..c43a3b8 100644
+--- a/node_modules/patch-package/dist/applyPatches.js
++++ b/node_modules/patch-package/dist/applyPatches.js
+@@ -85,7 +85,7 @@ function applyPatchesForApp(_a) {
+         })) {
+             // yay patch was applied successfully
+             // print warning if version mismatch
+-            if (installedPackageVersion !== version) {
++            if (installedPackageVersion !== version.replace(/\.dev$/, "")) {
```

Comment:
* This patch "solves" issue ds300/patch-package/issues/201 and replicates upstream PR ds300/patch-package/issues/224
* It was added to a new separate folder `'patches0'` instead of the existing usual `'patches'` folder so that `patch-package` can patch itself before being used to apply other patches (https://github.com/ds300/patch-package/issues/203#issuecomment-616187572). The zero at the end of `patches0` was meant to convey the idea that it comes first, at the very beginning, like `t0` in physics...